btree/btree.c

```diff
@@ -8582,8 +8621,13 @@ M0_INTERNAL int64_t m0_btree_lrulist_purge_check(enum m0_btree_purge_user user,
 	 * target watermark. For external user, purge lrulist till low watermark
 	 * or size whichever is higher.
 	 */
+	lru_purge_mode = lru_trickle_release ?
+			 M0_BTREE_PURGE_MODE_ON : M0_BTREE_PURGE_MODE_OFF;
 	size_to_purge = user == M0_PU_BTREE ?
-				(lru_space_used - lru_space_wm_target) :
+				(lru_purge_mode ?
```

Comment:
s/lru_purge_mode/lru_trickle_release/
'lru_trickle_release' variable name makes it more intuitive.